src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java

```diff
@@ -2,69 +2,69 @@
 
 import com.softserveinc.ita.dkrutenko.pageobjects.softserve.ContactUsPage;
 import com.softserveinc.ita.dkrutenko.pageobjects.softserve.MainPage;
+import com.softserveinc.ita.dkrutenko.pageobjects.softserve.usermodel.User;
+import com.softserveinc.ita.dkrutenko.pageobjects.softserve.usermodel.UserRepository;
 import com.softserveinc.ita.dkrutenko.utils.runners.TestRunner;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
-import static org.testng.Assert.assertEquals;
-import static org.testng.Assert.assertTrue;
+import static org.assertj.core.api.Assertions.assertThat;
+import static com.codeborne.selenide.Selenide.open;
 
 public class SoftServeTests extends TestRunner {
 
-    private final MainPage softServeBasePage = new MainPage();
+    private final MainPage softServeMainPage = new MainPage();
 
     @BeforeMethod
     private void openUrl() {
-        driver.get(softServeUrl);
+        open(softServeUrl);
     }
 
     @DataProvider
-    public Object[][] softServeLangVerification() {
+    public Object[][] softServeChangeLanguageVerification() {
         return new Object[][]{
                 {"We are advisors, engineers, and designers solving business challenges with innovative technology solutions",
                         "SoftServe berät Unternehmen weltweit in Digitalisierungs- prozessen und arbeitet am neuesten Stand der Technik",
                         "SoftServe - це провідна ІТ-компанія, що займається консалтингом та надає послуги у сфері цифрових технологій"}};
     }
 
-    @Test(dataProvider = "softServeLangVerification")
-    public void verifyLanguageSwitcher(String english, String german, String ukrainian) {
-        softServeBasePage.clickAcceptCookieMessageButton();
-        softServeBasePage.clickHeaderMenuButton();
-        softServeBasePage.clickLanguageSwitcher();
+    @Test(dataProvider = "softServeChangeLanguageVerification")
+    public void verifyChangeLanguageFunction(String english, String german, String ukrainian) {
+        softServeMainPage.clickAcceptCookieMessageButton();
+        softServeMainPage.clickHeaderMenuButton();
+        softServeMainPage.clickLanguageSwitcher();
         
-        var getTitle = softServeBasePage.getTitle();
+        var getTitle = softServeMainPage.getTitle();
         var expectedTitle = getTitle.contains(english)
                 || getTitle.contains(german)
                 || getTitle.contains(ukrainian);
-        assertTrue(expectedTitle);
+        assertThat(expectedTitle);
```

Comment:
assertJ should provide some method for such verifications
assertThat(expectedTitle)
.as("")
.contains(text1, text2, text3); <- take a look for such methods

Reply:
method .as doesn't work with contains

Reply:
Method contains doesn't work with boolean expectedTitle method. If i put getTitle in assert, i can use contains but, i cant realize how to verify all three text's. If i use this ".contains(text1, text2, text3);" -> test will fail,  because other 2 text's doesn't available on the page